### PR TITLE
Update telemetry.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated Playbooks telemetry terminology and updated section title.

### CODE REVIEW

**Comment by justinegeffen** [comment]
@agarciamontoro, I'm not sure whether I got the correct names for things so feel free to edit as needed. If there are new telemetry items, LMK, and I'll add them. :)

**Review by agarciamontoro — COMMENTED** [review]
The identifiers of the telemetry fields have not changed. We did add new ones, but the old ones keep the same terminology.

I can update the list here if you want me to, the source of truth is this file: https://github.com/mattermost/mattermost-plugin-playbooks/blob/master/server/telemetry/rudder.go

File: source/manage/telemetry.rst [hunk 1]
```diff
@@ -142,56 +142,56 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
...
-- ``Action``: The type of action performed against the incident: ``create``, ``end``, ``restart``, ``change_stage``, ``change_commander``, ``update_status``.
-- ``Public``: When creating an incident, ``true`` if it is public, and ``false`` if it is private.
+- ``OwnerUserID``: Unique identifier of the owner of the run.
+- ``TeamID``: Unique identifier of the team where the channel is created.
+- ``CreatedAt``: Timestamp of the run start.
+- ``PostID``: Unique identifier of the post from which the run was started (if relevant).
+- ``NumChecklists``: Number of stages in this run.
+- ``TotalChecklistItems``: Number of tasks in this run.
+- ``ActiveStage``: A number indicating the stage of the run (0-based).
+- ``Action``: The type of action performed against the run: ``start``, ``end``, ``restart``, ``change_stage``, ``change_owner``, ``update_status``.
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:169]:
```suggestion
- ``Action``: The type of action performed against the run: ``create``, ``end``, ``restart``, ``change_stage``, ``change_commander``, ``update_status``, ``add_timeline_event_from_post``, ``update_retrospective``, ``publish_retrospective``, ``remove_timeline_event``.
```

File: source/manage/telemetry.rst [hunk 2]
```diff
@@ -142,56 +142,56 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
...
+- ``Public``: When starting a run, ``true`` if it is public, and ``false`` if it is private.
 - ``ReminderTimerSeconds``: The next timer for the reminder to update the status, in seconds. It's tracked only when ``Action`` equals ``update_status``.
 
 **Data collected in tasks events**
 
-- ``IncidentID``: Unique identifier of the incident.
+- ``runID``: Unique identifier of the run.
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:175]:
```suggestion
- ``IncidentID``: Unique identifier of the run.
```

File: source/manage/telemetry.rst [hunk 3]
```diff
@@ -142,56 +142,56 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
...
 - ``UserID``: Unique identifier of the server.
-- ``Event``: Type of the event. There are three event types that are tracked: ``incident``, ``tasks``, ``playbook``.
+- ``Event``: Type of the event. There are three event types that are tracked: ``run``, ``tasks``, ``playbook``.
 - ``UserActualID``: Unique identifier of the user who initiated the action.
 
-**Data collected in incident events**
+**Data collected in run events**
 
-- ``IncidentID``: Unique identifier of the incident.
+- ``runID``: Unique identifier of the run.
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:160]:
```suggestion
- ``IncidentID``: Unique identifier of the run.
```

File: source/manage/telemetry.rst [hunk 4]
```diff
@@ -142,56 +142,56 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
...
-The following list details the types of Incident Collaboration metadata we collect:
+The following list details the types of Playbooks metadata we collect:
 
 **Data collected for all event types**
 
 - ``PluginVersion``: Version of the plugin.
 - ``ServerVersion``: Version of the server the plugin is running on.
 - ``UserID``: Unique identifier of the server.
-- ``Event``: Type of the event. There are three event types that are tracked: ``incident``, ``tasks``, ``playbook``.
+- ``Event``: Type of the event. There are three event types that are tracked: ``run``, ``tasks``, ``playbook``.
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:155]:
```suggestion
- ``Event``: Type of the event. There are three event types that are tracked: ``incident``, ``tasks``, ``playbook``.
```

File: source/manage/telemetry.rst [hunk 5]
```diff
@@ -155,43 +155,43 @@ The following list details the types of Incident Collaboration metadata we colle
...
-- ``TotalChecklistItems``: Number of tasks in this incident.
+- ``TotalChecklistItems``: Number of tasks in this run.
 - ``IsPublic``: ``true`` if the playbook is public, ``false`` if it is private.
 - ``NumMembers``: The number of members with access to this playbook.
 - ``NumSlashCommands``: The number of tasks with slash commands in this playbook.
 - ``ReminderTimerDefaultSeconds``: The default timer for the reminder to update the status, in seconds.
 - ``BroadcastChannelID``: Unique identifier of the channel where the status updates will be broadcasted.
 - ``UsesReminderMessageTemplate``: ``true`` if the playbook has a template for the reminder message, ``false`` otherwise.
-- ``Action``: The type of action performed against this playbook: ``create``, ``update``, ``delete``.
+- ``Action``: The type of action performed against this playbook: ``start``, ``update``, ``delete``.
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:194]:
```suggestion
- ``Action``: The type of action performed against this playbook: ``create``, ``update``, ``delete``.
```

**Comment by justinegeffen** [comment]
Thanks, @agarciamontoro! I've reverted the ones I changed and also added updated info. I imagine I missed a few other updates - if you're open to adding whatever I've missed to this list that'd be awesome. :)

File: source/manage/telemetry.rst [hunk 6]
```diff
@@ -155,36 +155,36 @@ The following list details the types of Incident Collaboration metadata we colle
...
-- ``CommanderUserID``: Unique identifier of the commander of the incident.
-- ``TeamID``: Unique identifier of the team where the incident channel is created.
-- ``CreatedAt``: Timestamp of the incident creation.
-- ``PostID``: Unique identifier of the post from which the incident was created (if relevant).
-- ``NumChecklists``: Number of stages in this incident.
-- ``TotalChecklistItems``: Number of tasks in this incident.
-- ``ActiveStage``: A number indicating the stage of the incident (0-based).
-- ``Action``: The type of action performed against the incident: ``create``, ``end``, ``restart``, ``change_stage``, ``change_commander``, ``update_status``.
-- ``Public``: When creating an incident, ``true`` if it is public, and ``false`` if it is private.
+- ``OwnerUserID``: Unique identifier of the owner of the run.
```
Comment (agarciamontoro) [inline source/manage/telemetry.rst:162]:
```suggestion
- ``CommanderUserID``: Unique identifier of the owner of the run.
```

**Review by agarciamontoro — COMMENTED** [review]
@justinegeffen, sorry  but I haven't had time to add the new info here :( 

I left a single comment of one field that was still changed, but otherwise looks good.

What do you think if we merge this and iterate when I'm back from my PTO?

**Comment by justinegeffen** [comment]
> What do you think if we merge this and iterate when I'm back from my PTO?

That works perfectly for me. Should I add it to the sprint board?

**Review by agarciamontoro — APPROVED** [review]
:tada:

**Comment by agarciamontoro** [comment]
> That works perfectly for me. Should I add it to the sprint board?

Sounds good!

**Pull request merged**